cloudify_docker/tasks.py

```diff
@@ -761,21 +721,21 @@ def create_container(ctx, docker_client, **kwargs):
 @handle_docker_exception
 @with_docker
 def start_container(ctx, docker_client, **kwargs):
-    container_args = \
-        ctx.node.properties.get(
-            'resource_config', {}).get('container_args', {})
+    resource_config = ctx.node.properties.get('resource_config', {})
+    container_args = resource_config.get('container_args', {})
     container = ctx.instance.runtime_properties.get('container', "")
     if not container:
         ctx.logger.info("container was not create successfully, nothing to do")
         return
     if not container_args.get("command", ""):
         ctx.logger.info("no command sent to container, nothing to do")
         return
-    ctx.logger.info(
+    ctx.logger.debug(
         "Running this command on container : {0} ".format(
             container_args.get("command", "")))
-    docker_client.start(container)
-    container_logs = follow_container_logs(ctx, docker_client, container)
+    container_obj = docker_client.containers.get(container)
+    container_obj.start()
```

Comment:
they moved the method to the model inner implementation , and since we already store the ID of the container , I need to do get [ and I think this method is not even mapped 😅  , but it is there , if anyone need to use stop and install ] 
I think I should add it to special  operation . what do you think @EarthmanT ?